docs/running_local.md

```diff
@@ -0,0 +1,82 @@
+> WARNING: This documentation is outdated, it will be updated soon
```

Comment:
Is it outdated?

Reply:
not now, let's keep this warning until prepare the full document